libs/libc/stdio/lib_libfflush.c

```diff
@@ -56,7 +56,7 @@
  *
  ****************************************************************************/
 
-ssize_t lib_fflush_unlocked(FAR FILE *stream, bool bforce)
+ssize_t lib_fflush_unlocked(FAR FILE *stream)
```

Comment:
Please remove the "bforce - flush must be complete." Input Parameter from lib_fflush Description

Reply:
Done.